### ISSUE TITLE
Making api calls in parallel

### ISSUE THREAD

**Issue by ravi160822** [pola-rs/polars#15603]
### Description

Please add a section to make api calls like get/post in parallel using the polars dataframe

**Comment by ravi160822** [comment]
that is what I am asking for a documentation on how to use polars + parallel api calls

**Comment by avimallu** [comment]
I think what @cmdlineluser is trying to say (and I agree) is that this isn't the right place to ask that question, because Polars doesn't directly deal with making `requests.get` calls. You're likely better off asking on Stackoverflow.

**Comment by deanm0000** [comment]
> (`httpx` also seems to be a newer http client that gets recommended these days)

+1 for httpx. It supports http2 for the odd site that requires it. Get to use the same library for sync and async. Syntax is mostly the same as requests.

Other than that this is definitely out of scope for polars documentation.
